woocommerce/woocommerce
# Correctly normalize "sold individually" product quantity in Checkout

This PR fixes an issue in which a product that's sold individually breaks Checkout if you have 2 of it in Cart.

### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Add an item twice to cart.
2. Edit the product so it's sold individually in its inventory settings.
3. Go to Checkout block, it should not break, and the item quantity should be 1.



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 
Fixes an edge case.
</details>
